activity.js

```diff
@@ -0,0 +1,114 @@
+"use strict";
```

Comment:
I have this in the root just like `decider` is in root. Should they both move to `lib`?